llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/Passes/Passes.h

```diff
@@ -225,6 +230,18 @@ std::unique_ptr<OperationPass<func::FuncOp>> createLinalgStrategyDecomposePass(
     const LinalgExt::LinalgTransformationFilter &filter =
         LinalgExt::LinalgTransformationFilter());
 
+/// Create a LinalgStrategyPeelPass.
+using LoopsToPeelComputationFunction = std::function<void(
+    OpBuilder &, Operation *, SmallVectorImpl<scf::ForOp> &)>;
+
+struct LinalgPeelOptions {
+  LoopsToPeelComputationFunction loopsToPeelComputationFunction = nullptr;
+};
+std::unique_ptr<OperationPass<func::FuncOp>> createLinalgStrategyPeelPass(
+    StringRef opName = "", const LinalgPeelOptions &opt = LinalgPeelOptions(),
+    const LinalgExt::LinalgTransformationFilter &filter =
+        LinalgExt::LinalgTransformationFilter());
```

Comment:
is this coming from a merging problem?

Reply:
Seems to be merging related. Will remove.